README.md

```diff
@@ -30,13 +30,28 @@ Tests can then be run after installation with:
 Call for Contributions
 ----------------------
 
-NumPy appreciates help from a wide range of different backgrounds.
-Work such as high level documentation or website improvements are valuable
-and we would like to grow our team with people filling these roles.
-Small improvements or fixes are always appreciated and issues labeled as easy
-may be a good starting point.
-If you are considering larger contributions outside the traditional coding work,
-please contact us through the mailing list.
+The NumPy project welcomes your expertise and enthusiasm!
+
+Small improvements or fixes are always appreciated. (Issues labeled as easy may be a good starting point.) If you are considering larger contributions to the source code, please contact us through the [mailing list](https://mail.python.org/mailman/listinfo/numpy-discussion) first. 
```

Comment:
"easy" -> "good first issue", fix line length
```suggestion
Small improvements or fixes are always appreciated. (Issues labeled as "good
first issue" may be a good starting point.) If you are considering larger
contributions to the source code, please contact us through the [mailing
list](https://mail.python.org/mailman/listinfo/numpy-discussion) first. 
```

Reply:
Also link to those issues:

https://github.com/numpy/numpy/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22